service/resource/namespace/current.go

```diff
@@ -4,7 +4,7 @@ import (
 	"context"
 
 	"github.com/giantswarm/microerror"
-	"github.com/giantswarm/operatorkit/framework/canceledcontext"
+	"github.com/giantswarm/operatorkit/framework/context/canceledcontext"
```

Comment:
Lets please try to do such vendor updates separately in the future. When you realize you have to do more than just "your stuff" try to boyscout in a separate PR and work from there. It is also that you mostly can just go ahead with the vendor PRs and get a cleaner PR with your "real changes" quite quickly.

Reply:
OK